app.py

```diff
@@ -1,12 +1,39 @@
-from flask import Flask
+import requests
+from http import HTTPStatus
+from flask import Flask, jsonify
 
-from api.post import post_api
+from api.dayimage import dayimage_api
+from api.errors import WrongCredentialsError
 
 app = Flask(__name__)
 
 app.config.from_object('config.Config')
 
-app.register_blueprint(post_api, url_prefix='/api/v1.0')
+app.register_blueprint(dayimage_api, url_prefix='/api/v1.0')
+
+
+@app.errorhandler(WrongCredentialsError)
```

Comment:
use please base error class here, because you can have more than one custom error in the future